its/ruling/src/test/resources/sonar-server/squid-S2095.json

```diff
@@ -0,0 +1,8 @@
+{
+'org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/platform/BackendCleanup.java':[
+67,
+114,
+142,
```

Comment:
This one the variable is given as a parameter, therefore I am not sure we should raise the issue of left open.